pkg/binding/bindingpolicy-resolution.go

```diff
@@ -49,13 +49,25 @@ type bindingPolicyResolution struct {
 	requiresSingletonReportedState bool
 }
 
-// objectData stores the UID, resource version, create-only bit,
+// ObjectData stores the UID, resource version, create-only bit,
 // and statuscollectors for an object.
-type objectData struct {
-	UID              string
-	ResourceVersion  string
-	CreateOnly       bool
+type ObjectData struct {
+	// UID is the UID of the object.
+	UID string
+	// ResourceVersion is the resource version of the object.
+	ResourceVersion string
+	// CreateOnly is a boolean that indicates whether the object was selected
+	// by at least one downsync clause with the createOnly bit set.
+	// If enabled, it means that the object should only be created and not
+	// maintained
+	CreateOnly bool
+	// StatusCollectors is a set of status collector names that are associated
 	StatusCollectors sets.Set[string]
+	// stalenessThresholdSecs dictates the staleness threshold for determining
+	// when the object's collected status is considered stale.
+	// If more than one BindingPolicy downsync clause selected the object,
```

Comment:
s/more than one BindingPolicy downsync clause selected the object/more than one resolution references the object/